#! /bin/sh /usr/share/dpatch/dpatch-run
## 553175_document_primary_uid_sign.dpatch by Daniel Leidert (dale) <daniel.leidert@wgdd.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Document the primary user id sign in --edit-key mode.
## DP:
## DP: <URL:http://bugs.debian.org/553175>
## DP: <URL:https://bugs.g10code.com/gnupg/issue1172>

@DPATCH@
diff -urNad trunk~/doc/gpg.texi trunk/doc/gpg.texi
--- trunk~/doc/gpg.texi	2009-09-02 14:31:17.000000000 +0200
+++ trunk/doc/gpg.texi	2010-01-22 02:27:18.888061353 +0100
@@ -845,7 +845,8 @@
 
 @c man:.RS
 The listing shows you the key with its secondary keys and all user
-ids. Selected keys or user ids are indicated by an asterisk. The trust
+ids. The primary user id is indicated by a dot, and selected keys or
+user ids are indicated by an asterisk. The trust
 value is displayed with the primary key: the first is the assigned owner
 trust and the second is the calculated trust value. Letters are used for
 the values:
